xbmc/osx/XBMCHelper.cpp

```diff
@@ -114,7 +114,7 @@ bool XBMCHelper::OnSettingChanging(const CSetting *setting)
     if (IsRunning() && GetMode() != remoteMode)
     {
       bool cancelled;
-      if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, 13146, 13147, -1, -1, cancelled, 10000))
+      if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, cancelled, -1, -1, 10000))
```

Comment:
hang on, this might not work. The mentioned method requires the first to parameters to be a localized string already and not only the string IDs. So we need to use a line1/2/3 version of this method with different parameter order :(
```
if (!CGUIDialogYesNo::ShowAndGetInput(13144, 13145, -1, -1, -1, -1, cancelled, 10000)
```
same parameter order required above then. We really have to adjust this dialog.